package.json

```diff
@@ -19,8 +19,8 @@
     "lint": "npm run lint:fix",
     "lint:fix": "npm run lint:typescript:fix && npm run lint:go:fix ",
     "lint:nofix": "npm run lint:typescript:nofix && npm run lint:go:nofix",
-    "lint:typescript:fix": "eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --fix .",
-    "lint:typescript:nofix": "eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue .",
+    "lint:typescript:fix": "eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --fix --report-unused-disable-directives .",
+    "lint:typescript:nofix": "eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --report-unused-disable-directives .",
```

Comment:
I was hoping this could go in `eslintrc` instead of command line arguments, but it didn't look like there was a good way to do so. (Adding more command line arguments is generally not great.)

Reply:
There's an eslint directive, but it only turns on warning-reporting (which we already have), but doesn't raise unused directives to error status.